context/app/static/js/components/cells/LogSliderWrapper/LogSliderWrapper.stories.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+
+import LogSliderWrapper from './LogSliderWrapper';
+
+export default {
+  title: 'Controls/LogSliderWrapper',
+  component: LogSliderWrapper,
+};
+
+export const LogSliderWrapperStory = (args) => {
+  const [logValue, setLogValue] = useState(1);
```

Comment:
I think this works for a single story, for components with more stories we'd likely want to use a decorator.  Looking at some of the state addons, I don't think an addon is something we need, but maybe I'm wrong.